Cargo.toml

```diff
@@ -33,10 +34,10 @@ itertools = "0.10.0"
 bincode = "1.3.1"
 serde = { version = "1.0", features = ["derive"] }
 sha2 = "0.9"
-blstrs = "0.4.0"
+blstrs = { default-features = true, version = "0.4.2", optional = true }
 pairing = "0.21"
 yastl = "0.1.2"
-ec-gpu = { version = "0.1.0"}
+ec-gpu = { version = "0.1.0", optional = true }
```

Comment:
`ec-gpu` contains just some trait definitions. Is this excluded from the `wasm` feature to reduce the build size?